src/storage/src/table/state_table.rs

```diff
@@ -422,7 +418,7 @@ impl<S: StateStore> StateTableRowIter<S> {
                 (Some(_), Some(_)) => {
                     // Throw the error.
                     cell_based_table_iter.next().await.unwrap()?;
-                    mem_table_iter.next().unwrap()?;
+                    mem_table_iter.next().unwrap();
```

Comment:
We may remove this line since there's no error to throw for memtable iterator now. :)

Reply:
I think it's seperate problem so I submit another PR. #3071

Reply:
Emmm think for twice and I think you are right.. It should be done in this PR. anyway..

Reply:
Actually previous it also can not be Error. It's always map with Ok.

Reply:
Previously there’re some decoding stuff in the `map` body so it can be error. Seems this has changed.😁